index.html

```diff
@@ -230,6 +221,19 @@ <h3>Example</h3>
       "addressLocality": "Nassau",
       "addressRegion": "New Providence",
       "addressCountry": "BS"
+    },
+    "permanentResidentCard": {
+      "type": "PermanentResidentCard",
+      "identifier": "83627465",
+      "name": "Permanent Resident Card",
+      "description": "Government of Example Permanent Resident Card.",
```

Comment:
I think these should be on the VC. They are needed for general rendering.

Reply:
Moved.  Defined them here for PRCCred though in VCDM v2 they are defined by default.  They could be added on the PRC itself as well if needed.